### ISSUE TITLE
MoPub 5.15 support

### ISSUE THREAD

**Issue by ab-zephyr** [liftoffio/LiftoffAds-iOS#7]
It seems the latest version of MoPub adapter (2.2.1) doesn't handle correct MPFullscreenAdAdapterDelegate in MoPub 5.15 for interstitials and rewarded videos.
As you know MoPub 5.15 introduces new method fullscreenAdAdapterAdDidDismiss in that delegate which should be called after fullscreenAdAdapterAdDidDisappear. Here is the sample how to call it

```
    // Signal that the fullscreen ad is closing and the state should be reset.
    // `fullscreenAdAdapterAdDidDismiss:` was introduced in MoPub SDK 5.15.0.
    if ([self.delegate respondsToSelector:@selector(fullscreenAdAdapterAdDidDismiss:)]) {
        [self.delegate fullscreenAdAdapterAdDidDismiss:self];
    }
```

**Comment by zdyn** [comment]
Thanks for opening this issue! We'll add this call and release an update shortly.

**Comment by zdyn** [comment]
We've released version 2.3.0 of our adapter which calls this new delegate. You can find the direct download [here](https://github.com/liftoffio/LiftoffAds-iOS/releases/tag/mopub-v2.3.0) or via CocoaPods. Thanks again and let us know if you run into any further issues.
